Sources/PT.PM.Common/TextUtils.cs

```diff
@@ -382,5 +384,64 @@ private static void ParseLineColumn(ReadOnlySpan<char> text, out int line, out i
                 throw new FormatException($"Invalid or too big column value {value} while {nameof(LineColumnTextSpan)} parsing.");
             }
         }
+
+        public static Literal CreateNumericLiteral(string value, TextSpan textSpan, int fromBase = 10)
+        {
+            switch (fromBase)
+            {
+                case 2:
+                    try
+                    {
+                        return new LongLiteral(Convert.ToInt64(value, 2), textSpan);
```

Comment:
Why don't we use `IntLiteral` for bases other than 10 (2, 8, 16)?

Reply:
There are very few languages out there allowing arbitrary base for integer literals. And I can't recall such language among more-or-less popular ones.